kubernetes/context.go

```diff
@@ -0,0 +1,187 @@
+package kubernetes
+
+import (
+	"io/ioutil"
+	"os"
+
+	"github.com/docker/docker/pkg/contextstore"
+	"github.com/pkg/errors"
+	"k8s.io/client-go/tools/clientcmd"
+	clientcmdapi "k8s.io/client-go/tools/clientcmd/api"
+)
+
+const (
+	kubernetesEndpoint  = "kubernetes"
+	serverKey           = "server"
+	skipTLSVerifyKey    = "skipTLSVerify"
+	defaultNamespaceKey = "defaultNamespace"
+	caFile              = "ca.pem"
+	certFile            = "cert.pem"
+	keyFile             = "key.pem"
+)
+
+// Context is a typed wrapper around a context-store context
+type Context struct {
+	Name             string
+	Server           string
+	SkipTLSVerify    bool
+	DefaultNamespace string
+}
+
+// LoadKubernetesConfig loads the kubernetes client config from the context
+func (c *Context) LoadKubernetesConfig(s contextstore.Store) (clientcmd.ClientConfig, error) {
```

Comment:
There's no (non-test) caller in this PR, even if I consider #1501 I only see a use in `runExport`. But nowhere seems to use it "for real" (so to speak). Did I miss it or is there a 3rd part somewhere?

Reply:
Was a bug, thanks